src/wasm/wasm-debug.cpp

```diff
@@ -635,6 +635,10 @@ struct LocationUpdater {
   }
 };
 
+static bool isTombstone(uint32_t x) {
+  return x == 0 || x == uint32_t(-1) || x == uint32_t(-2);
```

Comment:
I think that 0 shouldn't count as a tombstone anymore, right?

Reply:
I think we can still handle wasm emitted by older versions of LLVM by accepting 0. There seems to be no real downside to that?

Reply:
Maybe? the point of using a tombstone value instead of 0 is that 0 might end up being ambiguous in certain legitimate use cases. I don't know that we've actually hit any of those yet, but the debugger will certainly be expecting a tombstone value instead of 0 (that's what brought this up; I think they are going to land something upstream).

...which actually brings up another thing; this patch now makes Binaryen recognize different tombstone values, but shouldn't it also change what it outputs for those? Or are we just omitting entries entirely in that case? At least something is passing through to the debugger, else they wouldn't have noticed this in the first place.

Reply:
For the first paragraph, yes, good point. It does seem like maintaining compatibility has a risk of us hitting those cases. It does seem very likely to be disrupting in the short term to our users, though. How about leaving it as a TODO?

For the second paragraph, it should be fine. We either omit entries entirely, or leave them as is, in which case we'd still be using the same tombstone.

Reply:
OK, yeah we can leave this as a TODO and fix it once all the tools have gone through a few iterations. This is still new enough that I don't expect there to be a big backward compat problem.